src/xdocs/config_metrics.xml

```diff
@@ -197,6 +197,12 @@
                 Set, HashSet, SortedSet, TreeSet,
                 Map, HashMap, SortedMap, TreeMap</td>
           </tr>
+          <tr>
+            <td>excludeClassesRegexp</td>
+            <td>User-configured regular expression to ignore classes</td>
+            <td><a href="property_types.html#regexp">Regular Expression</a></td>
+            <td><code>&quot;&quot;</code>(empty)</td>
```

Comment:
```empty``` -> ```^$``` as for [LineLength](http://checkstyle.sourceforge.net/config_sizes.html#LineLength), for example.